TARS/src/mcu_main/sensors/LowGSensor.h

```diff
@@ -5,7 +5,7 @@
 #include "mcu_main/sensors/HighGSensor.h"
 #include "common/packet.h"
 #include "mcu_main/error.h"
-#include "mcu_main/hilsim/HILSIMPacket.h"
```

Comment:
Seems like the only diff in the sensor abstraction layer is this. Looks like the update() overload didn't change? That's a good thing but can someone reply to this to confirm that and that I'm not just tripping? Thanks